ci/Jenkinsfile2

```diff
@@ -5,12 +5,22 @@
 // DISCLAIMER: Code in this pipeline is currently "unwinded" for better readability of non-Jenkins users. //
 //                                                                                                        //
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////
-@Library('test-shared-library@1.2') _
+@Library('test-shared-library@1.3') _
 
 import ai.h2o.ci.Utils
+import ai.h2o.ci.buildsummary.StagesSummary
+import ai.h2o.ci.buildsummary.DetailsSummary
+import ai.h2o.ci.BuildResult
 
 def utilsLib = new Utils()
 
+BuildResult result = BuildResult.FAILURE
+
+// initialize build summary
+buildSummary('https://github.com/h2oai/datatable', true)
+// use default StagesSummary implementation
+buildSummary.get().addStagesSummary(this, new StagesSummary())
```

Comment:
Minor comment regarding API: 

why do you need to create addStagesSummary explicitly?

better would be:
```
buildSummary.get().createStagesSummary(this)
```